lang/fr/texts/discover.html

```diff
@@ -19,7 +19,7 @@ <h2 class="subheader">Base de données des aliments gratuits.</h2>
 
 <div class="row text-center">
 	<div class="large-12 columns">
-		<h2>Ouvrez votre nourriture et sachez ce que vous mangez</h2>
+		<h2>Ouvrez votre boîte de conserve pour savoir ce que vous mangez</h2>
 		<h4 class="subheader">Faites partir de notre base de données collaborative, libre, ouverte des produits alimentaires du monde entier!</h4>
```

Comment:
This was translated from English? I'd rather keep the version we have in production.

Reply:
c'est pour ça que j'ai pas mergé le français pour le moment